buildspec/linuxTests.yml

```diff
@@ -10,6 +10,7 @@ phases:
 
     build:
         commands:
+            - export AWS_TOOLKIT_TEST_USER_DIR=/tmp/
             - export AWS_TOOLKIT_TEST_NO_COLOR=1
```

Comment:
in hindsight, I don't know why these weren't placed in the env var section of buildspec

Reply:
I was also wondering

Reply:
fixed in https://github.com/aws/aws-toolkit-vscode/pull/1223